hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -257,61 +260,73 @@ public class OmMetadataManagerImpl implements OMMetadataManager,
   public static final String COMPACTION_LOG_TABLE =
       "compactionLogTable";
 
-  static final String[] ALL_TABLES = new String[] {
-      USER_TABLE,
-      VOLUME_TABLE,
-      BUCKET_TABLE,
-      KEY_TABLE,
-      DELETED_TABLE,
-      OPEN_KEY_TABLE,
-      MULTIPARTINFO_TABLE,
-      S3_SECRET_TABLE,
-      DELEGATION_TOKEN_TABLE,
-      PREFIX_TABLE,
-      TRANSACTION_INFO_TABLE,
-      DIRECTORY_TABLE,
-      FILE_TABLE,
-      DELETED_DIR_TABLE,
-      OPEN_FILE_TABLE,
-      META_TABLE,
-      TENANT_ACCESS_ID_TABLE,
-      PRINCIPAL_TO_ACCESS_IDS_TABLE,
-      TENANT_STATE_TABLE,
-      SNAPSHOT_INFO_TABLE,
-      SNAPSHOT_RENAMED_TABLE,
-      COMPACTION_LOG_TABLE
-  };
+  private class TableInitializer {
```

Comment:
Please add this at the bottom of the enclosing class to avoid breaking `FieldDeclarationsShouldBeAtStartOfClass` when it is enabled in HDDS-12907.

Reply:
@adoroszlai , thanks for reviewing this!  Moved it to the end of the file.